### DISCUSSION TITLE
Forge vs CurseForge type

### DISCUSSION THREAD

**Discussion by shadofall** [itzg/docker-minecraft-server#1845] [Q&A] [answered]
This might be a silly question. but what exactly is the difference between a Forge type and CurseForge type  I know there's diffrent settings but whats the underlying difference

**Comment by itzg** [comment, marked as answer]
Fair question since FORGE is a true server type whereas CURSEFORGE is a pseudo-type. And I'm pretty sure the "Forge" in "CurseForge" refers to the word [forge](https://www.google.com/search?q=define+forge) in the general sense. 

Using TYPE=FORGE gives you full control over the version of Forge and the mods you want applied. 

TYPE=CURSEFORGE basically unzips a modpack and hands over control to the start script included with the modpack.

**Reply by shadofall** [reply]
Many thanks.
